corehq/apps/data_dictionary/urls.py

```diff
@@ -16,6 +17,7 @@
 from corehq.apps.hqwebapp.decorators import waf_allow
 
 urlpatterns = [
+    path("case_count/<slug:case_type_name>", count_cases_for_case_type, name='count_cases_for_case_type'),
```

Comment:
Nit: Should an ending slash be added here?
```suggestion
    path("case_count/<slug:case_type_name>/", count_cases_for_case_type, name='count_cases_for_case_type'),
```